runtime/browser/xwalk_presentation_service_helper_android.cc

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2015 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "xwalk/runtime/browser/xwalk_presentation_service_helper.h"
+#include "xwalk/runtime/browser/xwalk_presentation_service_helper_android.h"
+
+namespace xwalk {
+
+
+DisplayInfoManagerServiceAndroid::DisplayInfoManagerServiceAndroid() {
+}
+
+DisplayInfoManagerServiceAndroid::~DisplayInfoManagerServiceAndroid() {
+}
+
+void DisplayInfoManagerServiceAndroid::FindAllAvailableMonitors(
+    std::vector<DisplayInfo>* info_list) {
+  if ( auto presentation_api = XWalkPresentationHost::GetInstance() ) {
```

Comment:
Yep, positive. This is the so called good habit to limit the scope of variable in the enclosed if-clause. It has pros and cons (e.g. causing this confusion), I didn't remember the specific reason why I wrote like this (kinda used to do something like this all the time).